tests/behat/features/manage-campaigns.feature

```diff
@@ -6,7 +6,8 @@ Feature: Manage campaigns
 
   Background:
     Given a "campaign" "Test Campaign" with "Description"="this is a test"
-      And I am logged in with "ADMIN" permissions
+      And the "group" "EDITOR" has permissions "Access to 'Pages' section" and "Access to 'Campaigns' section" and "Access to 'Files' section" and "FILE_EDIT_ALL"
```

Comment:
Are _all_ of these permissions required for _all_ of these scenarios? Seems likely that some of these can be split out to use less permissions.

Reply:
I created additional user with only "Campaign section" permissions.
But for the last `Scenario: Publishing a campaign publishes associated assets` I left "EDITOR" with permission as is.